src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -214,7 +214,7 @@ class NetworkOPsImp final
     void submitTransaction (Job&, STTx::pointer) override;
 
     void processTransaction (
-        Transaction::pointer transaction,
+        Transaction::pointer& transaction,
```

Comment:
I think this is unnecessary. Whether it's a pointer or reference, the underlying object still has changes to it reflected.